src/connector/src/parser/unified/debezium.rs

```diff
@@ -119,6 +129,16 @@ where
     fn access_field(&self, desc: &SourceColumnDesc) -> super::AccessResult {
         match self.op()? {
             ChangeEventOperation::Delete => {
+                // For delete events of MongoDB, the "before" and "after" field both are null in the value,
```

Comment:
Are you sure? According to the [official document](https://debezium.io/documentation/reference/stable/connectors/mongodb.html#mongodb-delete-events), the delete events should have non-null `before` payload. 

<img width="714" alt="Screenshot 2024-02-21 at 13 30 29" src="https://github.com/risingwavelabs/risingwave/assets/10192522/0440c109-99cc-4b4f-8a32-5ad29d5741da">

I am afraid you are talking about [Tombstone events](https://debezium.io/documentation/reference/stable/connectors/mongodb.html#mongodb-tombstone-events), which is emitted after the `delete` events:

> after Debezium’s MongoDB connector emits a delete event, the connector emits a special tombstone event that has the same key but a null value.

Reply:
I have turned off tombstone by setting `tombstones.on.delete` to `false` in `mongodb.properties`. And for a tombstone event, I think **the whole value** should be null instead of `before` and `after` fields are null.